miasm2/ir/ssa.py

```diff
@@ -1,6 +1,8 @@
 import miasm2.expression.expression as m2_expr
 from miasm2.expression.expression_helper import ExprDissector
-from miasm2.ir.ir import AssignBlock, irbloc
+from miasm2.ir.ir import AssignBlock
+
+from collections import deque
```

Comment:
(Not necessary to take in account) We try to keep an order in imports, [which is](https://www.python.org/dev/peps/pep-0008/#imports):

1. standard library imports (`collections`)
2. related third party imports
3. local application/library specific imports (`miasm2`)